core/src/main/java/org/kiwix/kiwixmobile/core/extensions/ActivityExtensions.kt

```diff
@@ -58,6 +65,45 @@ object ActivityExtensions {
     })
   }
 
+  fun Activity.openExternalUrl(
```

Comment:
This functionality is highly specific to an activity hosting a CoreReaderFragment so I would not make it something every activity can do